tracer/test/snapshots/OpenTelemetrySdkTests.SubmitsOtlpLogs.verified.txt

```diff
@@ -0,0 +1,133 @@
+﻿[
+  {
+    "resource_logs": [
+      {
+        "resource": {
+          "attributes": [
+            {
+              "key": "telemetry.sdk.name",
+              "value": {
+                "string_value": "sdk-name"
+              }
+            },
+            {
+              "key": "telemetry.sdk.language",
+              "value": {
+                "string_value": "dotnet"
+              }
+            },
+            {
+              "key": "telemetry.sdk.version",
+              "value": {
+                "string_value": "sdk-version"
+              }
+            },
+            {
+              "key": "service.name",
+              "value": {
+                "string_value": "unknown_service:dotnet"
```

Comment:
I think you'll need to have a separate OTel vs DD snapshot since the `service.name` (among others) may be different

Reply:
I've left one for now will update in a new PR with different tags and resource and different combinations. Otherwise I could addd the same service name, but I've added multiple scopes and fixed how they show up.